src/Control/SimpleResourceURLGenerator.php

```diff
@@ -135,17 +135,8 @@ protected function resolveModuleResource(ModuleResource $resource)
         $exists = $resource->exists();
         $absolutePath = $resource->getPath();
 
-        // Rewrite to _resources with public directory
-        if (Director::publicDir()) {
-            // All resources mapped directly to _resources/
-            $relativePath = Path::join(RESOURCES_DIR, $relativePath);
-        } elseif (stripos($relativePath ?? '', ManifestFileFinder::VENDOR_DIR . DIRECTORY_SEPARATOR) === 0) {
-            // If there is no public folder, map to _resources/ but trim leading vendor/ too (4.0 compat)
-            $relativePath = Path::join(
-                RESOURCES_DIR,
-                substr($relativePath ?? '', strlen(ManifestFileFinder::VENDOR_DIR ?? ''))
-            );
-        }
```

Comment:
Removed legacy logic for handling missing `public` folder. That folder was enforced in 5.0.0 so this is dead code.